nrf_modem/doc/at_interface.rst

```diff
@@ -154,6 +193,29 @@ The following snippet shows how to use the :c:func:`nrf_modem_at_cmd_async` func
 
    When there is a pending response, all other functions belonging to the AT API will block until the response is received in the callback function.
 
+.. note::
+   The application needs to escape characters in AT commands as it would when formatting it using :c:func:`printf`.
+   For example, the `%` needs to be escaped as `%%`.
+
+Differences with nrf_modem_at_printf
+====================================
+
+Both functions can be used to send a formatted AT command to the modem, the main difference is how the AT response is handled.
+:c:func:`nrf_modem_at_cmd` will will parse the modem AT response and return an error accordingly, in addition, it will copy the whole modem AT response to the supplied buffer.
```

Comment:
```suggestion
:c:func:`nrf_modem_at_cmd` will parse the modem AT response and return an error accordingly, in addition, it will copy the whole modem AT response to the supplied buffer.
```